### PR TITLE
Add composer package manager

### PR DESCRIPTION
**Pull request by SierraKomodo**
- [x] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**
- [X] **Have you checked that there aren't other open [pull requests](https://github.com/marticliment/WingetUI/pulls) for the same changes?**
- [ ] **Have you tested that the committed code can be executed without errors?**

Posting this up now as an unfinished draft as this is my first time working specifically with Python, and will probably need some assistance.

Immediately I do have one issue - Test builds appear to not be displaying any Composer entries with the current code, but also not crashing as far as I can tell, though I'm unsure where to find any raw/debug logs. So, two questions to start with:
1. What am I missing to have the installed packages tab include listings from the Composer class?
2. Where can the debug/raw logs be located, or how can I flag them to appear?

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (SierraKomodo) [inline .idea/codeStyles/Project.xml:1]: Self reminder to drop this commit before final approval.

**Comment by marticliment** [comment]
To register Composer, you will need to:
On wingetui/Interface/CustomWidgets/SpecificWidgets.py

you will need to add

```
from PackageManagers.composer import Composer
```


And add composer to the package managers list:
```
PackageManagersList: list[PackageManagerModule] = [
    Winget,
    Scoop,
    Choco,
    Pip,
    Npm,
    Dotnet,
    Composer
]

PackagesLoadedDict: dict[PackageManagerModule:bool] = {
    Winget: False,
    Scoop: False,
    Choco: False,
    Pip: False,
    Npm: False,
    Dotnet: False,
    Composer: False
}

StaticPackageManagersList: list[PackageManagerModule] = [
]

StaticPackagesLoadedDict: dict[PackageManagerModule:bool] = {
}

DynaimcPackageManagersList: list[DynamicPackageManager] = [
    Pip,
    Npm,
    Choco,
    Winget,
    Scoop,
    Dotnet,
    Composer
]

DynamicPackagesLoadedDict: dict[PackageManagerModule:bool] = {
    Pip: False,
    Npm: False,
    Winget: False,
    Choco: False,
    Scoop: False,
    Dotnet: False,
    Composer: False,
}

```

**Comment by marticliment** [comment]
You should be able to see the logs on "Package Manager Logs":
image

**Comment by Yousha** [comment]
Any updates on this PR?

**Comment by SierraKomodo** [comment]
> Any updates on this PR?

Life has struck in a not good way and I've had to backburner a lot of projects - This was one of them. I forgot to close the PR. Sorry :(

**Pull request closed without merging**